src/media/AppBadges.jsx

```diff
@@ -0,0 +1,52 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import cx from 'classnames';
+
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+
+export const IOS_DOWNLOAD_LINK = 'http://meetu.ps/2ZhShs';
+export const ANDROID_DOWNLOAD_LINK = 'http://meetu.ps/2ZhSyb';
+const getAppStoreDownloadPhoto = (platform, language) =>
+	// eslint-disable-next-line global-require, import/no-dynamic-require
+	require(`../../assets/images/app_download/${platform}/download_${language}.png`);
+
+/**
+ * Renders iOS and Android app badges
+ * @class AppBadges
+ */
+const AppBadges = ({ className, locale, getAppStorePhoto, ...other }) => {
```

Comment:
now that i think of it, it might be worth changing the prop name from `locale` to `language` to be extra clear that it's looking for somehting like `fr` instead of `fr-FR`